src/core/PseudoStash.pm6

```diff
@@ -74,17 +74,13 @@ my class PseudoStash is Map {
                 nqp::bindattr($stash, Map, '$!storage', nqp::ctxlexpad($ctx));
                 nqp::bindattr($stash, PseudoStash, '$!ctx', $ctx);
                 nqp::bindattr_i($stash, PseudoStash, '$!mode', PRECISE_SCOPE);
```

Comment:
Since I think really `MY::` is meant to just be like `OUTER::`, but without walking a step of the chain.